docs/website/blog/2024-03-25-reverse_etl_dlt.md

```diff
@@ -0,0 +1,196 @@
+---
+slug: reverse-etl-dlt
+title: "dlt adds Reverse ETL - build a custom destination in minutes"
+image:  https://storage.googleapis.com/dlt-blog-images/reverse-etl.png
+authors:
+  name: Adrian Brudaru
+  title: Open source Data Engineer
+  url: https://github.com/adrianbr
+  image_url: https://avatars.githubusercontent.com/u/5762770?v=4
+tags: [reverse etl, pythonic]
+---
+
+# Pythonic reverse ETL is here
+
+## Why python is the right approach for doing Reverse ETL
+
+Reverse ETL is generally about putting data into a business app. This data would often come from a SQL database used as a middle layer for data integrations and calculations.
+
+That’s fine - but nowadays most data people speak python, and the types of things we want to put into an operational app don’t always come from a DB, they often come from other business apps, or from things like a dataframe on which we did some scoring, etc.
+
+![reverse etl](https://storage.googleapis.com/dlt-blog-images/reverse-etl.png)
+
+### The full potential of Reverse ETL is in the flexibility of sources
+
+Sql databases are a good start, but in reality very often our data source is something else. More often than not, it’s a python analyst’s implementation of some scoring, or some business calculation.
+
+Other times, it’s a business app - for example, we might have a form that sends the response data to a webhook, from where it could end up in Salesforce, DWH and Slack as a notification. And of course, if this is done by a data person it will be done in Python.
+
+Such, it follows that if we want to cater to the data crowd, we need to be pythonic.
+
+## There’s synergy with ETL
+
+Reverse ETL is ultimately ETL. Data is extracted from a source, it’s transformer, and then loaded to a destination. The challenges are similar, the most notable difference being in the fact that pulling data from a strongly typed environment like a db and converting it to weakly typed JSON is MUCH easier than the other way around. So in fact you can argue that Reverse ETL is simpler than ETL.
+
+### Flavors of Reverse ETL
+
+Just like we have ETL and ELT, we also have flavors of Reverse ETL
+
+- Reverse ETL, or TEL: Transform the data to a specification, read it from db and send it to the app. An example could be.
```

Comment:
The preview for the list looks bland. Maybe make the heading bold.
- **Reverse ETL, or TEL:**
- **Tool Reverse ETL or ETL:**